docs/tutorials/index.md

```diff
@@ -71,17 +70,19 @@ The distribution directory contains `LICENSE` and `NOTICE` files and subdirector
 ## Start up Druid services
 
 Start up Druid services using the `auto` single-machine configuration.
-This configuration includes default settings that are appropriate for this tutorial, such as loading the `druid-multi-stage-query` extension by default so that you can use the MSQ task engine.
+This configuration includes default settings that are appropriate for this tutorial.
+For example, it loads `druid-multi-stage-query` extension so that you can use the MSQ task engine to ingest data using SQL.
 
-You can view that setting and others in the configuration files in the `conf/druid/auto`. 
+You can view the default settings in the configuration files located in `conf/druid/auto`.
 
 From the apache-druid-{{DRUIDVERSION}} package root, run the following command:
```

Comment:
Suggestion: use `apache-druid-<version>` or `apche-druid-$DRUID_VERSION`. The double brackets are a bit non-standard in this context, though they are used in Go's templating system.

Reply:
These variables get filled in when the docs get built, [for example](https://druid.apache.org/docs/latest/tutorials/):
> From the apache-druid-25.0.0 package root, run the following command: